packages/driver/cypress/integration/commands/actions/selectFile_spec.js

```diff
@@ -251,6 +259,32 @@ describe('src/cy/commands/actions/selectFile', () => {
         .then((input) => {
           expect(input[0].files[0].name).to.eq('valid.json')
           expect(input[0].files[1].name).to.eq('app.js')
+          expect(input[0].files[0].type).to.eq('application/json')
+          expect(input[0].files[1].type).to.eq('application/javascript')
+        })
+      })
+
+      it('allows users to override the inferred filenames and mimetypes', () => {
```

Comment:
This would make a really great example in the docs! (I don't know if you were going to do that or not)

Reply:
This is fairly similar to one of the existing examples:

https://github.com/cypress-io/cypress-documentation/blob/a07bac295906cdadff5241cc907603eded52c545/content/api/commands/selectfile.md#selecting-a-file-with-custom-filename-mimetype-and-lastmodified

For the docs examples, I tried to break them up a bit more, demonstrate one feature at a time; one example shows how to use an alias, another how to use a path, another haw to override fileName and mimeType. I could do one big "all the features at once" example though, if that seems like it would be helpful.

Reply:
I don't really have any examples around 'how to work with file inputs (eg, input.files[1].name, input.files[0].type) though, which might be what you're referring to? It wouldn't be bad to have - that's not Cypress functionality, it's just how browsers work, but no reason not to include a couple of example lines to help people get started.

Reply:
I actually really liked the combination of things, including doing a multiple-select with files specified inline, especially with one of those being an alias with overrides. It seemed like the "perfect storm" of API interactions :D

Reply:
I wasn't even thinking about the DOM API side of things, just the `.selectFile()` API here.